project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -75,59 +88,83 @@ public static void insertorUpdateEvent(Event event) {
+   *
    * @param eventId ID of event to be returned
-  */
+   */
   public static Optional<Event> getEventById(String eventId) {
     ResultSet resultSet =
         SpannerClient.getDatabaseClient()
             .singleUse()
             .executeQuery(
                 Statement.of(
                     String.format(
-                        "SELECT Name, Description, Labels, Location, Date, Time, Host,"
+                        "SELECT EventId, Name, Description, Labels, Location, Date, Time, Host,"
                             + " Opportunities, Attendees FROM %s WHERE EventID='%s'",
                         EVENT_TABLE, eventId)));
-    
+
     /** If ID does not exist */
     if (!resultSet.next()) {
       return Optional.empty();
     }
+    return Optional.of(shallowCreateEventFromDatabaseResult(resultSet));
+  }
 
-    // TO DO: replace with host from db, after PR #43 pushed
-    String NAME = "Bob Smith";
-    String EMAIL = "bobsmith@example.com";
-    User host = new User.Builder(NAME, EMAIL).build();
-    /* remove Hardcoded date */
-    Date date = Date.fromYearMonthDay(2016, 9, 15);
-    return Optional.of(
-        new Event.Builder(
-                /* name = */ resultSet.getString(0),
-               /* description = */ resultSet.getString(1),
-                /* labels = */ new HashSet<String>(resultSet.getStringList(2)),
-                /* location = */ resultSet.getString(3),
-                /* date = */ Date.parseDate(resultSet.getString(4)),
-                /* time= */ resultSet.getString(5),
-                /* host = */ host)
-            .build());
-    // TO DO: set volunteer opportunities, attendees by Querying those by ID, wait for PR 43, 44
+  /**
+   * Returns all events stored in DB
+   *
+   * @return Events with a shallow version of its host (no events attached to Users)
+   */
+  public static Set<Event> getAllEvents() {
+    Set<Event> events = new HashSet<>();
+    ResultSet resultSet =
+        SpannerClient.getDatabaseClient()
+            .singleUse()
+            .executeQuery(
+                Statement.of(
+                    String.format(
+                        "SELECT EventID, Name, Description, Labels, Location, Date, Host,"
+                            + " Opportunities, Attendees, Time FROM %s",
+                        EVENT_TABLE)));
+    while (resultSet.next()) {
+      Event event = shallowCreateEventFromDatabaseResult(resultSet);
+      events.add(event);
+    }
+    return events;
+  }
+
+  private static Event shallowCreateEventFromDatabaseResult(ResultSet resultSet) {
+    String eventId = resultSet.getString(0);
+    return new Event.Builder(
+            /* name = */ resultSet.getString(1),
+            /* description = */ resultSet.getString(2),
+            /* labels = */ new HashSet<String>(resultSet.getStringList(3)),
+            /* location = */ resultSet.getString(4),
+            /* date = */ resultSet.getDate(5),
+            /* time = */ resultSet.getString(9),
```

Comment:
for consistency can we always do time = resultSet.getString(6)? I always have time right after date. Will need change on line 143 of this file. I can also do it in a PR, lmk.

Reply:
Done